pkg/app/piped/executor/ecs/ecs.go

```diff
@@ -199,6 +199,11 @@ func runStandaloneTask(
 		return false
 	}
 
+	if !*ecsInput.RunStandaloneTask {
+		in.LogPersister.Infof("Skipped to the run task")
```

Comment:
```suggestion
		in.LogPersister.Infof("Skipped running task")
```
I think it would be clearer
because this if-clause means skipping RunTask, not skipping to RunTask. 
WDYT?